library/swift/test/HttpBridgeTests/ReceiveErrorTest.swift

```diff
@@ -0,0 +1,69 @@
+import Envoy
+import EnvoyEngine
+import Foundation
+import XCTest
+
+final class ReceiveErrorTests: XCTestCase {
+  func testReceiveError() throws {
+    // swiftlint:disable:next line_length
+    let apiListenerType = "type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager"
+    // swiftlint:disable:next line_length
+    let assertionFilterType = "type.googleapis.com/envoymobile.extensions.filters.http.assertion.Assertion"
+    let config =
+    """
+    static_resources:
+      listeners:
+      - name: base_api_listener
+        address:
+          socket_address:
+            protocol: TCP
+            address: 0.0.0.0
+            port_value: 10000
+        api_listener:
+          api_listener:
+            "@type": \(apiListenerType)
+            stat_prefix: hcm
+            route_config:
+              name: api_router
+              virtual_hosts:
+                - name: api
+                  domains:
+                    - "*"
+                  routes:
+                    - match:
+                        prefix: "/"
+                      direct_response:
+                        status: 503
```

Comment:
Would it make sense to document lower in the test that Envoy Mobile maps a 503 to an `EnvoyError` which is what's being tested?